packages/patternfly-4/react-core/src/components/Select/Select.tsx

```diff
@@ -292,8 +292,15 @@ class Select extends React.Component<SelectProps & InjectedOuiaProps, SelectStat
       return;
     }
 
-    const { children } = this.props;
-    const item = children.filter(child => child.props.value.toString() === value.toString())[0];
+    const { children, isGrouped } = this.props;
+    let item = null;
+    if (!isGrouped) {
+      item = children.filter(child => child.props.value.toString() === value.toString())[0];
```

Comment:
Could you test for `isGrouped`, and make the default case be the original `item = children.filter`... code?